styles/simple.scss

```diff
@@ -1,18 +1,106 @@
 .n-newsletter-signup--simple {
-	display: block;
+	display: flex;
+	flex-direction: row;
+	flex-shrink: 0;
 	border: 0;
+	gap: 20px;
+	padding-top: 20px;
+	background-color: oColorsByName("white-80");
+	padding-left: 24px;
+	padding-bottom: 0;
+
+	@include oGridRespondTo($from: M) {
+		align-items: start;
+		padding-left: 44px;
+		padding-bottom: 20px;
+	}
+
+	.n-newsletter-signup__card_content {
+		display: flex;
+		flex-direction: column;
+		align-items: start;
+		padding-left: 0;
+
+		@include oGridRespondTo($from: M) {
+			flex-direction: row;
+			align-items: center;
+		}
+
+	}
+
+	.n-newsletter-signup__body {
+		@include oTypographySans($scale: 0);
+		color: oColorsByName('black-80');
+		display: inline-block;
+		width: fit-content;
+
+		@include oGridRespondTo($from: M) {
+			width: 70%;
+		}
+	}
+
+	.n-newsletter-signup__top-meta {
+		top: 0;
+		right: 0;
+	}
+
+
+	.n-newsletter-signup__flex-child-left {
+		position: relative;
+		text-align: center;
+		color: oColorsByName('claret');
+		flex: 0 0 100%;
+		max-width: 158px;
+		max-height: 89px;
+
+		@include oGridRespondTo($from: M) {
+			max-width: 158px;
+			max-height: 89px;
+			flex: 0 0 30%;
+			overflow: hidden;
+		}
+
+		&__card_image {
+			max-width: 100%;
+		}
+	}
+
+	.n-newsletter-signup__heading {
+		@include oTypographyDisplay($scale: 2, $weight: 'bold');
+		
+		display: inline-block;
+		align-self: flex-end;
+		margin: 0;
+		font-size: 20px;
+		line-height: normal;
```

Comment:
**Change request:** Here we are overriding the styles set by `oTypographyDisplay` which we shouldn't be doing so we need to remove these styles and just use the ability to set font size via `oTypographyDisplay `